test/REQUIRE

```diff
@@ -1,2 +1,2 @@
 JuMP 0.16
```

Comment:
It's a bad idea to depend on JuMP for testing because all of the existing code will break with the 0.19 release.

Reply:
Yes that was my conclusion too, but this will require rethinking the tests, I had trouble understanding the mechanisms used: some vectors of solvers are defined before including the JuMP tests.
Maybe in another PR? Freezing the JuMP version to 0.18 upper bound for now

Reply:
We could also have a separate test suite, independent from JuMP. There are a couple of tests for CSIP (see [https://github.com/SCIP-Interfaces/CSIP/blob/master/test/test.c](test.c)) that cover a lot of features and would be easily ported, I guess.